### DISCUSSION TITLE
Is RPI Pico support for the ILI9486 parallel mode in the making?

### DISCUSSION THREAD

**Discussion by jadonmmiller** [Bodmer/TFT_eSPI#1233] [New processor/display support or capability request]
Unfortunately, I can confirm that there are a fair number of bugs, and I was wondering about an ETA for official support.

Thanks!

**Comment by destructor-c** [comment]
I can't confirm bugs yet, but io also like to get this pushed :) 
THANKS!

hey @jadonmmiller can you share code for your user_setup? i cant get it running?!
